gulpfile.babel.js

```diff
@@ -167,5 +169,26 @@ gulp.task('test', [
 // Run unit tests
 gulp.task('lint', [
   'lint:styles',
-  'lint:scripts'
+  'lint:scripts',
+  'lint:json'
 ])
+
+gulp.task('lint:json', () => {
+  gulp.src(['./app/data/*.json'])
+    .pipe(prettify())
+    .pipe(diff())
+    .pipe(diff.reporter({
+      quiet: false,
+      fail: true
+    }))
+})
+
+gulp.task('format', () => {
```

Comment:
Fair enough think it just formats `This little beautifier will reformat and reindent bookmarklets, ugly JavaScript, unpack scripts packed`. Think format reflects what it is doing making it more self explanatory.